projects/igniteui-angular/src/lib/grids/grid/grid-row.component.html

```diff
@@ -1,3 +1,12 @@
+
+ <ng-container *ngTemplateOutlet='addRow ? addTemp : defaultTemp'></ng-container>
+
+<ng-template #addTemp>
+    <div [className]='styleAddClasses' (webkitAnimationEnd)='animationEndHandler()' (transitionend)='animationEndHandler()'>
```

Comment:
This `<div>` should have its own class only to display flex its child elements. Currently, it has the same classes as its parent container `<igx-grid-row>`, so the bottom border is applied two times.